ftw/news/browser/resources/archive.scss

```diff
@@ -15,21 +19,9 @@
         display: block;
       }
     }
-
-    .count {
-      @include label-round();
-    }
   }
 
   .months {
     display: none;
-    .count {
-      &:before {
```

Comment:
These lines were just there for displaying the parentheses around the number of news per month. In https://github.com/4teamwork/ftw.news/commit/76bc80e2bd88bda478eb94ec976629d207bc37f6#diff-ceec225e4b24b8dacb84e3f56f1b8ef4L186 you can these parentheses were done in the backend. I don't think there will be any accessibility issues with these changes. Becuause the markup stays the same.